### PR TITLE
Eigen

### PR DESCRIPTION
**Pull request by Bart-CER**
I'm working on upgrading Dolfin, this is part of that process.

### CODE REVIEW

**Comment by wpoely86** [comment]
ok to test

File: easybuild/easyconfigs/e/Eigen/Eigen-3.2.2-goolf-1.5.14.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+version = '3.2.2'
+
+homepage = 'http://eigen.tuxfamily.org/index.php?title=Main_Page'
+description = """Eigen is a C++ template library for linear algebra:
+ matrices, vectors, numerical solvers, and related algorithms."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://bitbucket.org/%(namelower)s/%(namelower)s/get']
```
Comment (wpoely86) [inline easybuild/easyconfigs/e/Eigen/Eigen-3.2.2-goolf-1.5.14.eb]: use `BITBUCKET_SOURCE`

**Comment by boegel** [comment]
tiny remark to be handled; are you up for submitting a test report for this?

see https://github.com/hpcugent/easybuild/wiki/Review-process-for-contributions#automated-testing-of-easyconfigs-pull-requests for details on how to do this

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2095c92b2bb72bfeae32 for a full test report.

**Comment by boegel** [comment]
looking good, going in, thanks @Bart-CER!

**Pull request merged**